cupy/testing/array.py

```diff
@@ -74,12 +74,14 @@ def assert_array_max_ulp(a, b, maxulp=1, dtype=None):
         cupy.asnumpy(a), cupy.asnumpy(b), maxulp=maxulp, dtype=dtype)
 
 
-def assert_array_equal(x, y, err_msg='', verbose=True):
+def assert_array_equal(x, y, strides_check=False, err_msg='', verbose=True):
```

Comment:
The new argument should come last, because the order of arguments is a part of API.